sdk/ml/azure-ai-ml/azure/ai/ml/entities/_feature_store/feature_store.py

```diff
@@ -9,7 +9,7 @@
 from pathlib import Path
 from typing import Dict, Optional, Union
 
-from azure.ai.ml._restclient.v2023_04_01_preview.models import Workspace as RestWorkspace
```

Comment:
We need to keep workspace related changes in the preview version. Featurestore related changes are not in GA version yet, right?

Reply:
Bsed on my testing, the entities is OK to use the GA version, while the API, we will still use the original version for API call which defined in the ml_client.py:
https://github.com/Azure/azure-sdk-for-python/blob/1125bab611ae46450e62527e745da95a9881ab51/sdk/ml/azure-ai-ml/azure/ai/ml/_ml_client.py#L586C1-L592C10

Reply:
Though feature store operations (create/list/get/update) works well, but I compared the Workspace definition in two versions, there are some differences on showing up feature store settings. So i revert the change on this entity.